travis/cfg/travis_run_pylint_pr.cfg

```diff
@@ -11,7 +11,8 @@ cache-size=500
 [ODOOLINT]
 readme_template_url="https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst"
 manifest_required_author="Elico Corp"
-manifest_required_keys=license,support
+manifest_required_support="support@elico-corp.com"
```

Comment:
@victormartinelicocorp I'm not sure this is enough. You mean the check of value of the key is triggered by some naming convention? I would assume that somewhere, there's something to trigger this check on this field. IMO we should check all occurrences of `manifest_required_author` to see if there's a trigger and replicate that mechanism. If no other occurrence, then it means you're right about the automatic trigger based on name convention.